geotrek/feedback/models.py

```diff
@@ -331,13 +326,13 @@ def formatted_external_uuid(self):
         return formatted_external_uuid
 
     def notify_assigned_user(self, message):
-        subject = _("Geotrek - New report to process")
-        message = render_to_string("feedback/affectation_email.txt", {"report": self, "message": message})
+        subject = str(settings.EMAIL_SUBJECT_PREFIX) + str(_("New report to process"))
+        message = render_to_string("feedback/affectation_email.html", {"report": self, "message": message})
         self.try_send_email(subject, message)
 
     def notify_late_report(self, status_id):
-        subject = _("Geotrek - Late report processsing")
-        message = render_to_string(f"feedback/late_{status_id}_email.txt", {"report": self})
+        subject = str(settings.EMAIL_SUBJECT_PREFIX) + str(_("Late report processing"))
```

Comment:
concat string by using %s, .format() or f"strings". No need to cast string with str